data/strings/strings.txt

```diff
@@ -16958,6 +16958,98 @@
     zh-Hans = 关于 OpenStreetMap 的更多信息
     zh-Hant = 關於 OpenStreetMap 的更多資訊
 
+  [editor_osm_history]
+    tags = android
+    en = Your editing history
+    af = Jou redigeergeskiedenis
+    ar = سجل التحرير الخاص بك
+    az = Redaktə tarixçəniz
+    be = Ваша гісторыя рэдагавання
+    bg = Вашата история на редактиране
+    ca = El teu historial d'edició
+    cs = Vaše historie úprav
+    da = Din redigeringshistorik
+    de = Ihr Bearbeitungsverlauf
+    el = Το ιστορικό επεξεργασίας σας
+    es = Su historial de edición
```

Comment:
I suggest using the first person since the use of "usted" may infer third person, and the use of "tú" may not be natural for some native speakers:

"Mi historial de edición"